src/animate.jl

```diff
@@ -0,0 +1,57 @@
+"""
+Interpolations.jl requires that one(::Type{T}) be defined for any data
+type we want to interpolate. Rather than defining one(::Type{Vector}) here,
+which might have unforeseen consequences in other packages, we'll create
+a very simple wrapper type that just knows one() and *
+"""
+struct InterpolatableArray{A <: AbstractArray}
+    data::A
+end
+
+Base.one(::Type{InterpolatableArray{A}}) where {A} = 1
+Base.:*(n::Number, a::InterpolatableArray) = n * a.data
+
+rbd.normalize_configuration!(joint_type::JointType, q) = nothing
```

Comment:
I think these are unused now, right? Since you're using `RigidBodyDynamics.normalize_configuration!` below.

Reply:
good point. I'll fix that in #4